style/constants/color.js

```diff
@@ -12,3 +12,8 @@ export const borderCasing = `hsl(${hueBorderCasing}, 35%, 86%)`;
 export const parkFill = "hsl(136, 41%, 89%)";
 export const parkOutline = "hsl(136, 41%, 79%)";
 export const parkLabel = "hsl(136, 71%, 29%)";
+
+export const airportFill = "hsla(292, 41%, 89%, .3)";
+export const airportOutline = "hsl(292, 41%, 79%)";
+export const airportRunway = "hsl(270, 41%, 79%)";
+export const airportLabel = "hsl(270, 71%, 29%)";
```

Comment:
I think shifting the color hue more towards more of a bluish-purple makes the warmer purple boundary line more distinct while also leaving a distinction between gray roads and bluish-purple taxiways.

![Screen Shot 2022-05-13 at 11 10 38 PM](https://user-images.githubusercontent.com/281482/168408795-4703f0fd-3c80-4e5e-9a67-1656bae621b8.png)
![Screen Shot 2022-05-13 at 11 10 57 PM](https://user-images.githubusercontent.com/281482/168408798-7041b9db-9bc7-417e-9b73-3d1e4bb42c46.png)

Here's the code change:

```suggestion
export const airportFill = "hsl(250, 41%, 95%)";
export const airportOutline = "hsl(250, 41%, 79%)";
export const airportRunway = "hsl(250, 41%, 79%)";
export const airportLabel = "hsl(250, 71%, 29%)";
```

Moving the dotted boundary line (but not the purple casing) above everything else (except labels) also is an improvement I'd say, but this should probably be considered in a different PR to keep this one focused on airports.

![Screen Shot 2022-05-13 at 11 09 55 PM](https://user-images.githubusercontent.com/281482/168408966-89d30ea5-47f6-4e30-8c0a-a5109dd70e0e.png)